### PR TITLE
1.10.x

### PR DESCRIPTION
**Pull request by goulashify**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: java
+jdk:
+  - oraclejdk8
+before_install:
```
Comment (danigulyas) [inline .travis.yml]: Bla bla.

Comment (goulashify) [inline .travis.yml:4]: Bla bla!

File: README.md [hunk 2]
```diff
@@ -1,19 +1,37 @@
-### Why?
+[![Travis branch](https://img.shields.io/travis/Cosium/spring-data-jpa-entity-graph/1.10.x.svg)](https://travis-ci.org/Cosium/spring-data-jpa-entity-graph)
+[![Codecov branch](https://img.shields.io/codecov/c/github/Cosium/spring-data-jpa-entity-graph/1.10.x.svg)](https://codecov.io/gh/Cosium/spring-data-jpa-entity-graph)
+[![Maven Central](https://img.shields.io/maven-central/v/com.cosium.spring.data/spring-data-jpa-entity-graph.svg)](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.cosium.spring.data%22%20AND%20a%3A%22spring-data-jpa-entity-graph%22)
 
-Today, Spring Data JPA supports EntityGraph exlusively through annotations.  
+# Spring Data JPA EntityGraph
```
Comment (goulashify) [inline README.md:5]: Bla?

File: README.md [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
 
-1. Add the dependency :
+1. In addition to spring-data-jpa, add the library dependency :
     
     ```xml
     <dependency>
         <groupId>com.cosium.spring.data</groupId>
         <artifactId>spring-data-jpa-entity-graph</artifactId>
-        <version>1.11.00-SNAPSHOT</version>
+        <version>1.10.13</version>
```
Comment (goulashify) [inline README.md:34]: Supa fly! 🏄🏿‍♂️

**Review by goulashify — COMMENTED** [review]
Woo-woo! You would check one here, but i can't, since it's mine!

Here it would write it's accepted / requested changes.

**Pull request closed without merging**